Formula/a/asciinema.rb

```diff
@@ -18,7 +19,19 @@ class Asciinema < Formula
   depends_on "rust" => :build
 
   def install
+    ENV["ASCIINEMA_GEN_DIR"] = "generated"
+
     system "cargo", "install", *std_cargo_args
+
+    Dir["generated/man/*"].each do |manpage|
+      public_send(:"man#{File.extname(manpage).sub(/^\./, "")}").install manpage
+    end
+
+    bash_completion.install "generated/completion/asciinema.bash"
+    (share/"elvish/lib/asciinema.elv").install "generated/completion/asciinema.elv"
+    fish_completion.install "generated/completion/asciinema.fish"
+    pwsh_completion.install "generated/completion/_asciinema.ps1"
+    zsh_completion.install "generated/completion/_asciinema"
```

Comment:
Rearrange this and modify target name a bit.

```suggestion
    bash_completion.install "completion/asciinema.bash" => "asciinema"
    fish_completion.install "completion/asciinema.fish"
    zsh_completion.install "completion/_asciinema"
    pwsh_completion.install "completion/_asciinema.ps1" => "asciinema"
    (share/"elvish/lib").install "completion/asciinema.elv"
```